experimental/rust-tide/templates/default/.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
This might cause issues when importing projects into other IDE's (https://github.com/appsody/stacks/issues/126). Please could you remove the comments just in case?

Reply:
Removed with the new PR.